### DISCUSSION TITLE
Unknown recording date

### DISCUSSION THREAD

**Discussion by Valentina-Fed** [LAAC-LSCP/ChildProject#354] [Q&A] [answered]
Hi Lucas,

I have some recordings in the dataset without dates. I tried to put '0000-00-00', but it doesn't validate. Could you please provide an option 'unknown' also for this column? 

Thank you!

**Comment by alecristia** [comment, marked as answer]
can you please try one of the following:
- state the recording date as NA (which means not available)
- state the date of birth (DOB) as 1000-01-01 and then derive date of recording from that + age. so if DOB=1000-01-01 & child is 3 years 4 months, you do `1000-01-01 + (3*365.25 + 3*365.25/12)`
